src/main/java/hudson/plugins/gradle/injection/InjectionConfig.java

```diff
@@ -402,13 +426,21 @@ public FormValidation doCheckGradlePluginRepositoryUrl(@QueryParameter String va
 
     @Restricted(NoExternalUse.class)
     @POST
-    public FormValidation doCheckAccessKey(@QueryParameter String value) {
-        String accessKey = Util.fixEmptyAndTrim(value);
-        if (accessKey == null) {
+    public FormValidation doCheckAccessKeyCredentialId(@QueryParameter String value) {
```

Comment:
Need to check permissions https://github.com/jenkinsci/credentials-plugin/blob/master/docs/consumer.adoc#providing-a-ui-form-element-to-let-a-user-select-credentials

Reply:
Added the check